webapp/src/modules/account/sagas.spec.ts

```diff
@@ -129,3 +146,176 @@ describe('when handling the request to fetch account metrics', () => {
+      it('should signal that the request has failed with the request error', () => {
+        return expectSaga(accountSaga, catalystClient)
+          .provide([
+            [call([accountAPI, accountAPI.fetch], filters), { data: accounts }],
+            [
+              call([catalystClient, 'fetchProfiles'], addresses),
+              Promise.reject(new Error(error))
+            ]
+          ])
+          .put(fetchCreatorsAccountFailure(search, error))
+          .dispatch(fetchCreatorsAccountRequest(search))
+          .silentRun()
+      })
+    })
+  })
+
+  describe('when none of the requests fail', () => {
+    let search: string
+    let profiles: Profile[]
+    describe('and there is no search term', () => {
+      beforeEach(() => {
+        search = ''
+        filters = {
+          sortBy: AccountSortBy.MOST_COLLECTIONS
+        }
+        accounts = [
+          { address: 'address1' } as Account,
+          { address: 'address2' } as Account
+        ]
+        addresses = accounts.map(account => account.address)
+        profiles = [
+          { avatars: [{ ethAddress: addresses[0] }] } as Profile,
+          { avatars: [{ ethAddress: addresses[1] }] } as Profile
+        ]
+      })
+      it('should fetch the accounts with more collections using the accountAPI and their profiles using the catalyst lambdas', () => {
+        return expectSaga(accountSaga, catalystClient)
+          .provide([
+            [call([accountAPI, accountAPI.fetch], filters), { data: accounts }],
+            [call([catalystClient, 'fetchProfiles'], addresses), profiles]
+          ])
+          .put(
+            fetchCreatorsAccountSuccess(
+              search,
+              fromProfilesToCreators(profiles, accounts)
+            )
+          )
+          .dispatch(fetchCreatorsAccountRequest(search))
+          .silentRun()
+      })
+    })
+    describe('and there is a search term', () => {
+      let nftResults: NFTResult[]
+      beforeEach(() => {
+        search = 'a search term'
+        nftAPIFilters = {
+          category: NFTCategory.ENS,
+          search,
+          first: DEFAULT_FIRST_VALUE,
+          skip: DEFAULT_SKIP_VALUE
+        }
+        accounts = [
+          { address: 'address1' } as Account,
+          { address: 'address2' } as Account
+        ]
+        addresses = accounts.map(account => account.address)
+        filters = {
+          address: addresses,
+          sortBy: AccountSortBy.MOST_COLLECTIONS
+        }
+        nftResults = [
+          { nft: { owner: addresses[0] } } as NFTResult,
+          { nft: { owner: addresses[1] } } as NFTResult
+        ]
+        profiles = [
+          { avatars: [{ ethAddress: addresses[0] }] } as Profile,
+          { avatars: [{ ethAddress: addresses[1] }] } as Profile
+        ]
+      })
+      it('should fetch the ens that match the search term using the nftAPI and then and their profiles using the catalyst lambdas and accounts using the nftAPI', () => {
```

Comment:
Although we might have written the `its` wrongly, the expectation from the sagas is to eventually put an action, what do you think about adding this little peace of text?
```suggestion
      it('should fetch the ens that match the search term using the nftAPI and then and their profiles using the catalyst lambdas and accounts using the nftAPI and put the success action with the creators\' profiles', () => {
```